backend/api/tasks/statistics.py

```diff
@@ -0,0 +1,110 @@
+import datetime
+from flask_restful import Resource, current_app, request
+from backend.services.stats_service import NotFound, StatsService
+
+
+def validate_date_input(date):
+    if date:
+        date_format = "%Y-%m-%d"
+        current_date = datetime.datetime.now()
+        date = datetime.datetime.strptime(date, date_format)
+        if date <= current_date:
+            return date
+        raise ValueError("Date out of range for task activity")
+
+
+class TasksStatisticsAPI(Resource):
+    def get(self):
+        """
+        Get Task Stats
+        ---
+        tags:
+          - tasks
+        produces:
+          - application/json
+        parameters:
+            - in: query
+              name: start_date
+              description: Date to filter as minimum
+              required: true
+              type: string
+              default: null
+            - in: query
+              name: end_date
+              description: Date to filter as maximum
+              required: true
```

Comment:
Are you planning to restrict the maximum date range we can query? Like 1 or 2 years? I think that param doesn't need to be required in case the current date is inside that range. It would make the frontend interface behaviour better to the users.